portal-web/test/functional/com/liferay/portalweb/tests/enduser/workflow/WorkflowMetrics.testcase

```diff
@@ -0,0 +1,95 @@
+@component-name = "portal-workflow"
+definition {
+
+    property portal.release = "true";
+    property portal.upstream = "true";
+    property testray.main.component.name = "Workflow";
+
+    setUp {
+        TestCase.setUpPortalInstance();
+
+        User.firstLoginPG();
+
+        Navigator.openURL();
+
+        ProductMenu.gotoPortlet(
+            category = "Workflow",
+            panel = "Control Panel",
+            portlet = "Metrics");
+    }
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			ProductMenu.gotoPortlet(
+				category = "Workflow",
+				panel = "Control Panel",
+				portlet = "Metrics");
+
+            WorkflowMetrics.goToWorkflowSLASettings(workflowProcessName = "Single Approver");
+
+			if (IsElementPresent(locator1 = "WorkflowMetricsSLA#SLA_TABLE")) {
+				WorkflowMetrics.tearDown();
```

Comment:
hey Dennys, what do you think we encapsulate the WorkflowMetrics.goToWorkflowSLASettings() and the **IF** clause and the to the WorkflowMetrics.tearDown() ? So we will have only the call for the tearDown() here.